images/kcp-registrar/register.sh

```diff
@@ -124,7 +124,7 @@ register() {
         else
             printf "Registering cluster\n"
             KUBECONFIG=${kcp_kcfg} kubectl kcp workload sync "${clusters[$i]}" \
-                 --syncer-image ghcr.io/kcp-dev/kcp/syncer:release-0.4 \
+                 --syncer-image ghcr.io/kcp-dev/kcp/syncer:KCP_BRANCH \
```

Comment:
Could we use an env variable instead? The container should have the variable set when the image is built, and that it would allow to run the script locally without having to modify it.

Reply:
Good idea, will try